src/frame.js

```diff
@@ -0,0 +1,54 @@
+function Frame(multipliers, isFinalFrame) {
```

Comment:
I like that you're encapsulating the fact that this is the final frame within the frame (rather than treating them as separate classes).

I think it's possible not to even need a special case though :). For me, the frame's responsibility is to allow up to two rolls to knock over up to 10 pins, and then to accept bonus scores if applicable. This doesn't change if it's the last frame - what does change is that further rolls only count towards the bonus and not towards another frame. This feels like the game's responsible to manage.